WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -120,19 +115,23 @@ public void loadPost() {
         protected Spanned doInBackground(Void... params) {
             Spanned contentSpannable;
 
-            if (mActivity == null || mActivity.getPost() == null) {
+            if (getActivity() == null) {
                 return null;
             }
 
-            Post post = mActivity.getPost();
+            PostModel post = ((EditPostActivity) getActivity()).getPost();
```

Comment:
Would be a bit better to pass the PostModel from the Activity to the Fragment, makes the fragment reusable.